src/cr/cube/cubepart.py

```diff
@@ -775,9 +775,14 @@ def pairwise_significance_tests(self):
 
     @lazyproperty
     def population_counts(self):
-        return (
-            self.table_proportions * self._population * self._cube.population_fraction
+        proportions = (
+            self.row_proportions
+            if self._dimensions[0].is_cat_date
+            else self.column_proportions
+            if self._dimensions[1].is_cat_date
+            else self.table_proportions
```

Comment:
I don't know if checking the dimension type this way is the best approach.
1. at the `_Slice` level we already have `_rows_dimension` and `_columns_dimension` so it would be more explicit instead of `self._dimensions[0] or [1]`
2. I wonder if the cat date type can be a real dimension type and so do the check like:
```
if self.rows_dimension_type == DT.CAT_DATE:
```
maybe in here https://github.com/Crunch-io/crunch-cube/blob/a1b615593281a9cdcdb8c87e708bccaabe139422/src/cr/cube/dimension.py#L275

Reply:
what about MR × date? is the date as columns still dimension 1 if there's a selections dimension?

Reply:
no, @malecki in that case it's hidden, because `_dimensions` only has the visible ones. But I agree with the first @ernestoarbitrio 's comment, I'll make it more explilcit. About the special type, though, @ernestoarbitrio I thought about that too, but I'm afraid what other things it might break, since it's definitely a CAT dimension, and needs to behave like one. And since we're doing it with enums, and not classes, I can't simply inherit from the base class and have mostly the same behavior... Anyways, I'll check...

Reply:
ok i see

Reply:
Ok, I ended up refactoring it according to your recommendation. It seems to be working fine with exporter too...